__tests__/app.test.js

```diff
@@ -32,6 +32,38 @@ describe('GET', () => {
 
     });
 
+    describe('GET: /api', () => {
+
+        describe('GOOD PATH:', () => {
+
+            test(`200: /api responds with an object containing all available endpoints in this API.
+            each endpoint key should have a object property which contains endpoint desccription,
+            any queries the enddpoint takes, and an example response`, () => {
+                return request(app)
+                .get('/api')
+                .expect(200)
+                .then(({body}) => {
+                    const {endpoints} = body;
+                    expect(typeof endpoints).toBe('object');
+                    expect(Array.isArray(endpoints)).toBe(false);
+                    expect(endpoints !== null).toBe(true);
```

Comment:
This could be simpler and also a more robust test if we had simply required endpoints.js into the test suite and expected endpoints toEqual requiredInEndpoints